farm/modeling/language_model.py

```diff
@@ -128,7 +132,37 @@ def load(cls, pretrained_model_name_or_path, n_added_tokens=0, language_model_cl
             language_model = cls.subclasses[config["name"]].load(pretrained_model_name_or_path)
         else:
             if language_model_class is None:
-                language_model_class = cls.get_language_model_class(pretrained_model_name_or_path)
```

Comment:
We should keep the get_language_model_class() call here and rather add your changes in that method